packages/expo-cli/src/credentials/credentials.ts

```diff
@@ -121,15 +121,22 @@ export type FcmCredentials = {
   fcmApiKey: string;
 };
 
+export type Keystore = Android.Keystore;
+
 export type AndroidCredentials = {
   experienceName: string;
-  keystore: Android.Keystore | null;
+  keystore: Keystore | null;
   pushCredentials: FcmCredentials | null;
 };
 
 export const keystoreSchema: CredentialSchema<Android.Keystore> = {
   id: 'keystore',
   name: 'Android Keystore',
+  provideMethodQuestion: {
```

Comment:
Is there a reason why the keystore generation prompts are written this way instead of the usual 
```
Will you provide your own <thing>?
- Let Expo handle the process
- I want to upload my own file
```
It would simplify the code if we just used the normal prompts which accomplish the same thing, unless there is good reason to use custom prompts

Reply:
A lot of people don't understand what it means to "let expo handle it" and assume that there is a way to automatically generate new keystore that will work when uploading what often leads to need to reset keystore via google support.